### PR TITLE
fix: Social Feed input minimized on mobile

### PR DESCRIPTION
**Pull request by WaDadidou**

### CODE REVIEW

**Review by n0izn0iz — DISMISSED** [review]
LGTM

File: packages/components/socialFeed/NewsFeed/NewsFeed.tsx [hunk 1]
```diff
@@ -100,28 +105,42 @@ export const NewsFeed: React.FC<NewsFeedProps> = ({
         >
           <Header />
         </View>
-        <SpacerColumn size={2.5} />
+        {/*<SpacerColumn size={2.5} />*/}
```
Comment (hthieu1110) [inline packages/components/socialFeed/NewsFeed/NewsFeed.tsx:108]: Can we remove this if it's not used ?

**Review by hthieu1110 — DISMISSED** [review]
LGTM,

**Comment by zxxma** [comment]
LGTM 👍

**Pull request merged**